operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -72,6 +71,27 @@ spec:
 # Choose "anonymous" to allow full access to Kiali without requiring any credentials.
 # Choose "openshift" to use the OpenShift OAuth login which controls access based on
 # the individual's  RBAC roles in OpenShift. Not valid for non-OpenShift environments.
+# Choose "ldap" to enable LDAP based authentication. There are additional configurations for
+# under ldap in the yaml file. The configurations for LDAP are as follows
+#    ---
+#    strategy: ldap
+#    ldap:
+#       # AD IP host and port
+#       ldap_host: example.com
```

Comment:
This kiali_cr.yaml file is used for documentation - all values shown here are the actual defaults. So make sure you put the defaults in here. If you want to further explain a setting, please add additional comments above it in the same format and style of comments you see elsewhere in this file.

In addition, all settings documented here should be listed alphabetically.

Based on those two things, this section should look something like:

```
#       ldap_attributes: ['memberOf', 'cn', 'mail']
#
# Base attributes all users must have. For example, "DC=example,DC=com".
#       ---
#       ldap_base: ""
#
# The binding that determines a user. For example, "CN={USERID},OU=xyz,OU=Users,OU=Accounts,DC=example,DC=com".
#       ---
#       ldap_bind_dn: ""
#
#       ldap_group_filter: "(cn=%s)"
#       ldap_host: ""
#       ldap_insecure_skip_verify: false
#       ldap_port: 0
#
# The role filter can be used to filter the user roles using regular expressions. For example, ".*xyz.*".
#       ---
#       ldap_role_filter: ""
#
#       ldap_search_filter: "(&(name={USERID}))"
#       ldap_use_ssl: false
#       ldap_user_filter: "(cn=%s)"
```